Sitefinity CLI/Templates/11.1/ResourcePackage/Bootstrap/README.md

```diff
@@ -0,0 +1,207 @@
+    </div>
+    <div class="sf_colsIn col-md-6" data-sf-element="Column 2" data-placeholder-label="Column 2">
+    </div>
+</div>
+```
+**IMPORTANT:** Do **not** remove `sf_colsIn` since it is a system CSS class that indicates where a placeholder is created.
+
+To create a new grid widget is just as simple:
+1. Duplicate an existing grid widget template.
+2. Name the new file, so that the new grid widget is displayed in the list of grid widgets in the *Layout* tab in the Page editor.
+
+**Example:** To create a simple placeholder with `<section>` tag, create `section.html` in `/ResourcePackages/Bootstrap/GridSystem/Templates/`
+```
+	<section class="section" data-sf-element="Section" data-placeholder-label="Section">
+	</section>
+```
+
+## Recommendations for managing widget templates
+
+### Modify widget templates
+
+To make upgrades easier, we recommend not to change the default widget templates. If you need to make changes to a default template, we recommend that you create a new one by duplicating the existing template you want to modify. You then make the changes on the newly created template.
+
+## Responsive design ##
+
+The responsiveness of widgets, including grid widgets, relies solely on the Bootstrap 3 framework.
+
+The only exception is the Navigation widget. You have the option to transform the navigation into a `<select>`.
+
+To do this, in the Navigation widget template file, for example, `/ResourcePackages/Bootstrap/MVC/Views/Navigation/NavigationView.Horizontal.cshtml`, you need to use the `@Html.Action("GetView", new { viewName = "Dropdown",  model= Model})` helper method.
+The helper method renders the `<select>`. It is commented out by default but you can use it in combination with the responsive utility classes of Bootstrap.
+To use the helper method, modify its markup in the `/ResourcePackages/Bootstrap/MVC/Views/Navigation/Dropdown.cshtml` file.
+
+## Location of project frontend assets
+All project-specific frontend assets like SCSS, images, JavaScript files, fonts, and so on need to be placed in the `assets/src/project` folder. When the Grunt task is run, all source files are processed and moved to the `assets/dist` folder, from which they are used in the project.
+
+### SCSS
+Place all your SCSS files in the `assets/src/project/sass` folder. We recommend that you create subfolders to organize the project files and only then import the files in `main.scss`
+**Example:**
+```
+File structure
+|- sass
+|-- settings
+|--- _colors.scss
+|--- _typography.scss
+...
+|-- base
+|--- _link.scss
+|--- _typography.scss
+...
+
+main.scss
+//Import Bootstrap from npm
+@import "../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap.scss";
+@import "../../../../node_modules/magnific-popup/src/css/main.scss";
+
+// Sitefinity
+@import "../../sitefinity/sass/components/icons/sf-icon-font";
+@import "../../sitefinity/sass/widgets/socialShare/sf-sprite";
```

Comment:
do we still need this? @JenPeleva

Reply:
@Novkirishki @jordanilchev We don't need this. Could you please get the latest templates.